src/components/NavBar/NavBar.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import LoginNavBar from './LoginNavBar';
+import DefaultNavBar from './DefaultNavBar';
+import { useSelector } from 'react-redux';
+let NavBar = ({ searchParams }) => {
```

Comment:
please add line break after the imports for better readability

Reply:
fixed